proposals/0370-modifiers.rst

```diff
@@ -54,37 +54,40 @@ Proposed Change Specification
    the Haskell 2010 Report for all BNF syntax, and recall that we use braces
    to denote "zero or more")::
 
-     type     ::= btype [ {modifier} -> type ]
-     prefix%  ::= '%'    -- only in prefix position
-     modifier ::= prefix% atype
+     type          ::= btype [ {modifier} -> type ]
```

Comment:
Since `modifiers` is defined anyway, one could make this (very minor) simplification
```suggestion
     type          ::= btype [ modifiers -> type ]
```